src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,80 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String account = new String();
```

Comment:
too complicated
your code is too much c++ style :)

Reply:
))))

Reply:
your code is too much c++ style
What does it mean?